src/routes/1st/index.svelte

```diff
@@ -0,0 +1,322 @@
+  <section id="welcome">
+    <h2 class="headers">Welcome to ACM!</h2>
+
+    <p>
+      acmCSUF provides a social online platform, Discord, that brings passionate Titans together to
+      engage in multiple conversations regarding our community, the tech field, and many more. Join
+      <a href="/discord">our Discord server</a> to learn more about future events, meet new people, and
+      get involved with the community!
+    </p>
+  </section>
+
+  <section id="fork">
+    <h2 class="headers">Fork <code>acmcsuf.com</code></h2>
+
+    <p dir="auto">
+      <img
+        width="300"
+        src="https://camo.githubusercontent.com/fcf9a4ed664cc63de2fcb14d1135072ba6d4c74a8e9bdb224ad6ab1e72600c3b/68747470733a2f2f6669727374636f6e747269627574696f6e732e6769746875622e696f2f6173736574732f526561646d652f666f726b2e706e67"
+        alt="fork this repository"
+        data-canonical-src="https://firstcontributions.github.io/assets/Readme/fork.png"
+        style="max-width: 100%;"
+      />
+    </p>
+
+    <p>
+      Fork the repository by clicking on the fork button on the top of <a
+        href="https://github.com/ethanthatonekid/acmcsuf.com#readme"
+        ><code>acmcsuf.com</code>'s page on GitHub</a
+      >. This will create a copy of this repository in your account.
+    </p>
+  </section>
+
+  <section id="clone">
+    <h2 class="headers">Clone the repository</h2>
+
+    <p dir="auto">
+      <img
+        width="300"
+        src="https://camo.githubusercontent.com/4c3f7f1bec4f04db40ecf58dc2e19c2d8992f100f3bbbc4767a9d20b29f4a43d/68747470733a2f2f6669727374636f6e747269627574696f6e732e6769746875622e696f2f6173736574732f526561646d652f636c6f6e652e706e67"
+        alt="clone this repository"
+        data-canonical-src="https://firstcontributions.github.io/assets/Readme/clone.png"
+        style="max-width: 100%;"
+      />
+    </p>
+
+    <p dir="auto">
+      Now clone the forked repository to your machine. Go to your GitHub account, open the forked
+      repository, click on the code button and then click the <em>copy to clipboard</em> icon.
+    </p>
+
+    <p dir="auto">
+      Open a terminal and run the following git command: <code>git clone "url you just copied"</code
+      > where "url you just copied" (without the quotation marks) is the url to this repository (your
+      fork of this project). See the previous steps to obtain the url.
+    </p>
+
+    <p>
+      <img
+        width="300"
+        src="https://camo.githubusercontent.com/1c0cf8056422ff414eee75142b213c5970e085c2e33c0a6d69dc2639d98216f1/68747470733a2f2f6669727374636f6e747269627574696f6e732e6769746875622e696f2f6173736574732f526561646d652f636f70792d746f2d636c6970626f6172642e706e67"
+        alt="copy URL to clipboard"
+        data-canonical-src="https://firstcontributions.github.io/assets/Readme/copy-to-clipboard.png"
+        style="max-width: 100%;"
+      />
+    </p>
+
+    <p>For example:</p>
+    <pre><code>
+git clone https://github.com/this-is-you/acmcsuf.com.git
```

Comment:
not mobile responsive as it does not word wrap or overflow: scroll

Reply:
https://user-images.githubusercontent.com/8981287/189439801-1b1f9276-63da-4234-a9ee-2a1ca5d1ef31.mp4